docs/setup/sso.md

```diff
@@ -68,6 +68,59 @@ If there is only one upstream provider configured and the local password databas
 
 This section contains sample configurations for popular OIDC providers.
 
+### Authelia
+
+These instructions assume that you have already enabled the OIDC provider support in [Authelia](https://www.authelia.com/).
+
+Add a client for MAS to the Authelia configuration:
+
+```yaml
+  - client_id: "<client-id>" # TO BE FILLED
```

Comment:
Could you include where this is exactly in the config?
Something like:
```yaml
identity_providers:
  oidc:
    clients:
      - client_id: "<client-id>" # TO BE FILLED
```

And maybe with a link to the relevant documentation: https://www.authelia.com/configuration/identity-providers/openid-connect/clients/